### DISCUSSION TITLE
how do i solve 412 Gmail_API: Invalid To header error

### DISCUSSION THREAD

**Discussion by ssenyondo67** [emailjs-com/help#13] [Q&A]
I keep getting the status code 412 response of Gmail_API: Invalid To header error if I send a message from my app.
but I receive a test email from the dashboard I can't understand why

**Comment by xr0master** [comment]
In this case, the error message describes the issue very well. The recipient address is not correct, it has to be an email address.

**Comment by ssenyondo67** [comment]
the error appears only when using the local host cause I have created a test project with the same code and deployed it on Vercel and it is working fine still am using the same details, it also worked on[ Fiddle shell](https://jsfiddle.net/api/post/library/pure/)
)
